string switch_to_mode
---
bool switched